pkg/controllers/builder.go

```diff
@@ -374,6 +378,21 @@ func makeContainers(vdb *vapi.VerticaDB, sc *vapi.Subcluster) []corev1.Container
 	return cnts
 }
 
+// getEnvVars returns a list of EnvVars from the annotations
+// in the CR
+func getEnvVars(vdb *vapi.VerticaDB) []corev1.EnvVar {
```

Comment:
Suggest a name like `translateAnnotationsToEnvVars`.  I think `gentEnvVars` is too generic and doesn't say what is actually happening.

Reply:
Yes it sounds better.